langchain/retrievers/vectara_retriever.py

```diff
@@ -0,0 +1,36 @@
+"""Milvus Retriever"""
+from typing import Optional, List
+
+from langchain.schema import BaseRetriever, Document
+from langchain.vectorstores.vectara import Vectara
+
+class VectaraRetriever(BaseRetriever):
```

Comment:
we probably want to make Vectara.as_retriever() return this - so i would move it into that file (similar to how we do with redis)

Reply:
so just move VectaraRetriever to vectara.py and get rid of vectara_retriever.py altogether?

Reply:
yep!

Reply:
Perfect. Done!